src/PurgeClient/VarnishPurgeClient.php

```diff
@@ -7,20 +7,30 @@
 namespace EzSystems\PlatformHttpCacheBundle\PurgeClient;
 
 use FOS\HttpCacheBundle\CacheManager;
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
 
 /**
  * Purge client based on FOSHttpCacheBundle.
  */
 class VarnishPurgeClient implements PurgeClientInterface
 {
+    public const INVALIDATE_TOKEN_NAME_PARAM = 'http_cache.varnish_invalidate_token_name';
+    public const INVALIDATE_TOKEN_PARAM = 'http_cache.varnish_invalidate_token';
```

Comment:
Side note (maybe you're aware of this already) : this bundle is also used with eZ Platform 1.13, which means PHP 5.6: access modifiers for const can't be used 😛